### ISSUE TITLE
Muting video stops system wide audio

### ISSUE THREAD

**Issue by pyby** [SRGSSR/srgmediaplayer-apple#3]
A user of the library reports this issue: once a video is muted, the audio is muted globally.

Steps to reproduce:
1) Start music e.g. in Spotity or the music app
2) Run app that uses SRGMediaPlayer and programatically mute the sound ( playerController.muted = YES )

What happens:
All sounds are muted, incl. the sound of other apps.

Expected:
Sound of media player is muted, but Spotify continues to play

Tested with version: 
SRGMediaPlayer 1.9.9
OS: iOS9.3.2

How play a muted video without impacting other audio apps?

Thanks

**Comment by defagos** [comment]
TL;DR: This is not a player issue, but merely an audio setup management issue. 

Your application audio is most probably non-mixable when playing your inline content. This happens because your application is using a non-mixable category like `AVAudioSessionCategorySoloAmbient`, or `AVAudioSessionCategoryPlayback` with default options when navigating the news timeline. For more information about the available categories, you should have a look at the [official documentation](https://developer.apple.com/library/ios/documentation/AVFoundation/Reference/AVAudioSession_ClassReference/#//apple_ref/doc/constant_group/Audio_Session_Categories).

In your case, you must therefore switch between different audio session setups when appropriate:
- When your application audio must be mixable with other contents (e.g. when navigating the news timeline with muted videos), use a mixable category like `AVAudioSessionCategoryAmbient`:

```
[[AVAudioSession sharedInstance] setCategory:AVAudioSessionCategoryAmbient withOptions:0 error:nil];
```

or a non-mixable category like `AVAudioSessionCategoryPlayback` with proper options (in your case `AVAudioSessionCategoryOptionMixWithOthers`):

```
[[AVAudioSession sharedInstance] setCategory:AVAudioSessionCategoryPlayback withOptions:AVAudioSessionCategoryOptionMixWithOthers error:nil];
```
- When your application audio must not be mixable (e.g. when watching a video with sound), use a non-mixable category like `AVAudioSessionCategoryPlayback`  with proper options:

```
[[AVAudioSession sharedInstance] setCategory:AVAudioSessionCategoryPlayback withOptions:0 error:nil];
```

How exactly you must choose your audio session categories and options, and when you need to change them, depend on your application needs. The [official audio session documentation](https://developer.apple.com/library/ios/documentation/Audio/Conceptual/AudioSessionProgrammingGuide/Introduction/Introduction.html) is a good start to understand everything you can achieve, but getting a perfect behaviour can be tricky.

In our application, for example, we usually setup the audio session category when the `RTSMediaPlaybackStateReady` state is reached, and reset it in some cases when playback ends. We also tweak sessions to restore audio from other apps, listen to notifications to properly deal with phone call interruptions, etc. There is a lot of potential in this area.
